### ISSUE TITLE
Installation instructions

### ISSUE THREAD

**Issue by danbettles** [welovecoding/editorconfig-netbeans#82]
Hello

Could you let me know how to install the plugin, please?  I'm a web developer and I use the HTML5 + PHP-flavoured version of NetBeans.  I've read the installation instructions in your wiki but they don't mean much to me - I've never installed a downloaded plugin before and I've never developed one either - and NetBeans' plugin installer doesn't recognise any of the files in the project.  

Many thanks,
Dan

**Comment by ZaLiTHkA** [comment]
Those installation instructions seem to be aimed more at building and running the plugin from source, while it sounds like you're simply trying to install a release version.

In this case:

1. Download the `.nbm` file for the latest version from the [Releases page](https://github.com/welovecoding/editorconfig-netbeans/releases)
2. Open NetBeans, select Tools > Plugins
3. Switch to "Downloaded" tab and click "Add Plugins..."
4. Browse to and select the `.nbm` you downloaded
5. Click "Install"

If all went well, NetBeans should then prompt you to restart and activate the plugin. Everything else from this point will work based on `.editorconfig` files in your project folder. Keep in mind you'll need to make sure NetBeans itself is running with Java 8 in order for this to work.

Hope that helps.. :)

**Comment by bennyn** [comment]
@ZaLiTHkA - I could not have put it better. Thank you very much!
@danbettles - Just let me know if there are still uncertainties.

**Comment by danbettles** [comment]
Thank you very much, @ZaLiTHkA; that did help.

Both: would you like me to add those instructions to the Wiki because they should be beneficial to others.

Thanks again.

**Comment by ZaLiTHkA** [comment]
Only a pleasure. I'm all for the idea of sharing knowledge and I've been doing software testing and writing user manuals for about 7 years now, so it's practically second nature for me these days.

To be honest, I think this project's wiki could actually do with a bit of TLC.. I'd be quite happy to build a nice wiki structure for you guys if you would like? This is something I'd have to do outside of my working time though (8:00am - 4:30pm, GMT+2).

**Comment by bennyn** [comment]
@danbettles: Good idea! We will add it to our wiki.
@ZaLiTHkA: We would be really happy if you would like to work on our wiki. Are you already allowed to edit it or do you need extra permissions?

**Comment by ZaLiTHkA** [comment]
No need to change settings, I just checked now and I can edit the wiki.

The default options for GitHub projects is to allow anyone to edit the wiki pages, unless you select the "Restrict editing to collaborators only" option in the project settings.

**Comment by bennyn** [comment]
Cool! Then you are ready to go. :dancer: Feel free to edit it. Will mention you then in our collaborators section. :+1:

**Comment by ZaLiTHkA** [comment]
I got a bit distracted this evening, so unfortunately I got to this a bit later than I was planning to, however I have extended the [plugin installation](../wiki/Installation) instructions a bit.

I'll go through all the pages over the next couple days, creating links between them and removing consolidating information where applicable (such as the "Plugin Development" and "Releasing" pages).

@bennyn, if you're happy with the installation instructions, then I think this issue can be closed. :)
